src/test/cypress/cypress/integration/UserCard.spec.js

```diff
@@ -190,6 +190,30 @@ describe('User Card ', function () {
     })
   })
 
+  describe('Test spinners', function () {
+    it('Check spinner appears when card template is loading', () => {
+
+      cy.delayRequestResponse('/businessconfig/**/*');
+
+      cy.loginOpFab('operator1_fr', 'test');
+      cy.get('#opfab-navbarContent').find('#opfab-newcard-menu').click();
```

Comment:
check here the spinner as well 
   cy.checkLoadingSpinnerIsDisplayed();
    cy.checkLoadingSpinnerIsNotDisplayed();